t/Util.pm

```diff
@@ -698,4 +699,36 @@ sub get_tracer {
     return $read_trace;
 }
 
+sub run_picotls_client {
+    my($opts) = @_;
+    my $port = $opts->{port}; # required
+    my $host = $opts->{host} // '127.0.0.1';
+    my $path = $opts->{path} // '/';
+    my $cli_opts = $opts->{opts} // '';
+    my $connection = $opts->{keep_alive} ? "keep-alive" : "close";
+
+    my $cli = bindir() . "/picotls/cli";
+    die "picotls-cli ($cli) not found" unless -e $cli;
+
+    my $tempdir = tempdir();
+    my $cmd = "exec $cli $cli_opts $host $port > $tempdir/resp.txt 2>&1";
+    diag $cmd;
+    open my $fh, "|-", $cmd
+        or die "failed to invoke command:$cmd:$!";
+    autoflush $fh 1;
+    print $fh <<"EOT";
+GET $path HTTP/1.1\r
+Host: $host:$port\r
+Connection: $connection\r
```

Comment:
I'm not sure if we need to support keep-alive when the code below only can wait for the server to close the connection at the end of the HTTP response?

Reply:
At first, I'd like to use this new utility in `t/40tls13-early-data.t` as well, but it would be too complex if a single subroutine supported both single-request mode and multiple-request mode with keep-alive. Now, this utility only supports single-request mode, so this tweak should be removed.